03/app.js

```diff
@@ -41,18 +40,58 @@ books.getAuthor = function(isbn) {
     return false;
 }
 
-books.getTitle = function(isbn, lang) {
+books.getTitle = function (isbn, lang) {
 
+    // check if input isbn exist in object
+    if (typeof this[isbn] === 'undefined') {
+        return null;
+    }
+
+    const title = this[isbn]['title'];
+
+    for (const key in title) {
+        if (key === lang) {
+            return title[key];
+        }
+    }
```

Comment:
Zdecydowanie przesadzone. Wyobraź sobie, że masz 1000 elementów w obiekcie. Zamiast od razu pobrać wskazany tj. `title[lang]` to wykonujesz w najgorszym wypadku 999 porównań ;)

Reply:
@devmentor-pl, rzeczywiście – po Twoim komentarzu i z "czystą głową" spojrzałem na to dzisiaj i teraz zupełnie inaczej spojrzałem na ten problem. Poprawione ;).